assets/scripts/datalayer/common.js

```diff
@@ -192,7 +192,9 @@ document.shopAnalytics = {
     .on('click', '.products .product a', onProductClick)
     .on('click', '.single_add_to_cart_button', onProductAddToCart)
     .on('click', '.remove_from_cart_button, .woocommerce-cart-form .product-remove > a, .cart_item td.product-remove .remove', onRemoveSingleProduct)
-    .on('click', 'th.product-remove .remove', onEmptyCart);
+    .on('click', 'th.product-remove .remove', onEmptyCart)
+    .on('click', 'nav a', onNavItemClick)
```

Comment:
Same same here.

Reply:
As you can read from the ticket, the goal is to track click on nav item links. A consideration was to use the landmark `nav[role=navigation]`. Whenever you using `nav` the `role="navigation"` is implied already and thus redundant. Therefore I decided to use `nav` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#Description

> Using the <nav> element will automatically communicate a section has a role of navigation. Developers should always prefer using the correct semantic HTML element over using ARIA.